packages/lib-core/src/runtime/PluginStore.ts

```diff
@@ -155,18 +150,19 @@ export class PluginStore implements PluginStoreInterface {
 
     Array.from(this.loadedPlugins.entries()).forEach(([pluginName, plugin]) => {
       entries.push({
-        pluginName,
         status: 'loaded',
-        metadata: plugin.metadata,
+        pluginName,
+        metadata: _.pick(plugin.manifest, ['name', 'version', 'dependencies']),
+        customProperties: plugin.manifest.customProperties,
```

Comment:
@florkbr Right now, we're just passing `manifest.customProperties` through to `entry.customProperties`.

Would it be better to add `?? {}` fallback here to avoid `entry.customProperties` being `undefined` ?